web/static/css/base/_grid-settings.scss

```diff
@@ -1,6 +1,12 @@
-$small-screen: new-breakpoint(max-width em(540));
-$medium-screen: new-breakpoint(max-width em(720));
-$large-screen: new-breakpoint(max-width em(860));
+@function px-to-em($value) {
+  $em-base: 16;
 
-$small-screen-up: new-breakpoint(min-width em(520));
-$medium-screen-up: new-breakpoint(min-width em(720));
+  @return ($value / $em-base) * 1em;
+}
+
+$small-screen: new-breakpoint(max-width px-to-em(540));
```

Comment:
I'm not sure if there's a benefit to this function. Why not just use pixels for the breakpoints? That would solve the deprecation issue and not require any additional functions.

Reply:
i.e. `$small-screen: new-breakpoint(max-width 540px);`

Reply:
👍 yeah that sounds much better. I'll update this.